mpp/include/mpp_util.inc

```diff
@@ -1389,10 +1388,10 @@ end function rarray_to_char
                    call mpp_error(FATAL, 'get_ascii_file_num_lines: Error reading line '//trim(text)// &
                         ' in file '//trim(FILENAME)//'.')
                 end if
-                if ( len_trim(str_tmp) == LENGTH ) then
-                   write(UNIT=text, FMT='(I5)') length
+                if ( len_trim(str_tmp) == FMS_FILE_LEN ) then
+                   write(UNIT=text, FMT='(I5)') FMS_FILE_LEN
```

Comment:
I think we want to revert this to the original code here as the LENGTH here is not referring to a file length or pathname.